test/sqllogictest/object_ownership.slt

```diff
@@ -2713,7 +2713,8 @@ COMPLETE 0
 simple conn=mz_system,user=mz_system
 DROP OWNED BY materialize;
 ----
-db error: ERROR: cannot drop "materialize.public.t" without CASCADE: still depended upon by non-owned catalog items "materialize.public.v"
+db error: ERROR: cannot drop table "materialize.public.t": still depended upon by "materialize.public.v"
```

Comment:
Can you add a test case where it's depended on by multiple objects?

Reply:
Done